modules/ROOT/pages/timeseries.adoc

```diff
@@ -0,0 +1,220 @@
+
+[source, teradata-sql]
+----
+REPLACE VIEW NYC_taxi_trip_ts as
+SELECT
+$TD_TIMECODE_RANGE time_bucket_per
+,vendor_id
+,sum(passenger_count) passenger_cnt
+,sum(trip_time_in_secs) avg_trip_time_in_secs
+FROM TRNG_NYCTaxi.trip 
+GROUP BY TIME(MINUTES(15) and vendor_id)
+USING TIMECODE(pickup_datetime)
+WHERE extract(month from pickup_datetime)=11;
+----
+
+Let's calculate a 2 hours moving average on our 15-minutes time series. 2 hour is 8 * 15 minutes periods.
+
+[source, teradata-sql]
+----
+SELECT TOP 10 * FROM MovingAverage (
+  ON NYC_taxi_trip_ts PARTITION BY vendor_id ORDER BY time_bucket_per
+  USING
+  MAvgType ('S')
+  WindowSize (8)
+  TargetColumns ('passenger_cnt')
+) AS dt 
+WHERE begin(time_bucket_per)(date) = '2013-11-10'
+ORDER BY vendor_id, time_bucket_per;
+----
+
+
+Result:
+
+----
+
+time_bucket_per	                                                      vendor_id	    passenger_cnt   avg_trip_time_in_secs   passenger_cnt_smavg
+-----------------------------------------------------------------     ---------     -------------   ---------------------   -------------------
+2013-11-10 00:15:00.000000-05:00,2013-11-10 00:30:00.000000-05:00     CMT           5949            3075511                 2187.625
+2013-11-10 00:30:00.000000-05:00,2013-11-10 00:45:00.000000-05:00     CMT           5810            2918201                 2913.75
+2013-11-10 00:45:00.000000-05:00,2013-11-10 01:00:00.000000-05:00     CMT           5566            2767533                 3609.375
+2013-11-10 01:00:00.000000-05:00,2013-11-10 01:15:00.000000-05:00     CMT           5358            2676729                 4277.25
+2013-11-10 01:15:00.000000-05:00,2013-11-10 01:30:00.000000-05:00     CMT           5133            2524393                 4907.5
+2013-11-10 01:30:00.000000-05:00,2013-11-10 01:45:00.000000-05:00     CMT           4896            2411368                 5403.125
+2013-11-10 01:45:00.000000-05:00,2013-11-10 02:00:00.000000-05:00     CMT           4815            2276216                 5448.75
+2013-11-10 02:00:00.000000-05:00,2013-11-10 02:15:00.000000-05:00     CMT           4699            2267700                 5278.25
+2013-11-10 02:15:00.000000-05:00,2013-11-10 02:30:00.000000-05:00     CMT           4505            2126760                 5097.75
+2013-11-10 02:30:00.000000-05:00,2013-11-10 02:45:00.000000-05:00     CMT           4355            2031154                 4915.875
+
+----
+
+==== Note: 
+In addition to above time series operations, Vantage also provides a special time series tables with Primary Time Index (PTI). These are regular Vantage tables with PTI defined rather than a Primary Index (PI). Though tables with PTI are not mandatory for time series functionality/operations, PTI optimizes how the time series data is stored physically and hence improves performance considerably compared to regular tables. 
```

Comment:
Asciidoc supports admonitions that are visually distinct. I would replace this fragment with:
```suggestion
NOTE: In addition to above time series operations, Vantage also provides a special time series tables with Primary Time Index (PTI). These are regular Vantage tables with PTI defined rather than a Primary Index (PI). Though tables with PTI are not mandatory for time series functionality/operations, PTI optimizes how the time series data is stored physically and hence improves performance considerably compared to regular tables. 
```